# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/PBrowser.ipdl
# Commit: 6709a3bc1a1c
# Full Hash: 6709a3bc1a1c4ac18dce0f6db78efe12b3e1f178
# Author: Kashav Madan <kmadan@mozilla.com>
# Date: 2019-07-15 21:39:46
# Regressor Bug: 1543251
# File Overlap Count: 1
# Description:
#   Bug 1543251 - Move hasBeforeUnload from PBrowser to PWindowGlobal, r=nika
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D37003
# ==============================================================================

diff -r 83881815f007 -r 6709a3bc1a1c dom/ipc/PBrowser.ipdl
--- a/dom/ipc/PBrowser.ipdl	Mon Jul 15 11:01:04 2019 +0000
+++ b/dom/ipc/PBrowser.ipdl	Mon Jul 15 17:30:26 2019 +0000
@@ -578,8 +578,6 @@
 
     async AccessKeyNotHandled(WidgetKeyboardEvent event);
 
-    async SetHasBeforeUnload(bool aHasBeforeUnload);
-
     async RegisterProtocolHandler(nsString scheme, nsIURI handlerURI, nsString title,
                                   nsIURI documentURI);
 